src/atc_pyspark.py

```diff
@@ -0,0 +1,33 @@
+from pyspark.sql import SparkSession
+from pyspark.sql.types import *
+import pyspark.sql.functions as F
+from pyspark.sql.types import (
+    ArrayType, LongType, StringType, StructField, StructType)
+
+spark = SparkSession.builder.getOrCreate()
+df = spark.read.format('xml').options(rowTag='drug').load('s3n://rxminer/drugbank/drugbank.xml')
+df.createOrReplaceTempView("input_table")
+df.show(10)
```

Comment:
you probably might not need this .show function, unless it was to show the output the data for an easier user experience when they run this code